doc/source/cluster/aws-tips.rst

```diff
@@ -71,3 +71,244 @@ You should see something like
      region                <not set>             None    None
 
 Please refer to `this discussion <https://github.com/ray-project/ray/issues/9327>`__ for more details.
+
+
+.. _aws-cluster-cloudwatch:
+
+Using Amazon CloudWatch
+-----------------------
+
+Amazon CloudWatch is a monitoring and observability service that provides data and actionable insights to monitor your applications, respond to system-wide performance changes, and optimize resource utilization.
+CloudWatch integration with Ray requires an AMI (or Docker image) with the Unified CloudWatch Agent pre-installed.
+
+AMIs with the Unified CloudWatch Agent pre-installed are provided by the Amazon Ray Team, and are currently available in the us-east-1, us-east-2, us-west-1, and us-west-2 regions.
+Please direct any questions, comments, or issues to the `Amazon Ray Team <https://github.com/amzn/amazon-ray/issues/new/choose>`_.
+
+.. list-table:: `Unified CloudWatch Agent Images <https://github.com/amzn/amazon-ray>`_
```

Comment:
In the rendered page this doesn't look like a table caption, but rather like a dangling link just sitting there. Could we maybe make this table caption read "All available unified CloudWatch agent images" and then have a full sentence linking to the `amazon-ray` repo after?

Reply:
@Zyiqin-Miranda can you give me a ping after this and I'll merge?

Reply:
Thanks @maxpumperla for the review! Updated table name without the link.

Reply:
Sure, thanks @wuisawesome, updated and looks like passing ci tests